Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 name = "norad"
-version = "0.2.0"
+version = "0.2.1"
 authors = ["Colin Rofls <colin@cmyr.net>", "Nikolaus Waxweiler <madigens@gmail.com>"]
 license = "MIT/Apache-2.0"
 edition = "2018"
 keywords = ["font", "ufo", "fonts"]
-repository = "https://github.com/linebender/norad"
-description = "Read and write Unifed Font Object files."
+repository = "https://github.com/mfeq/norad"
+description = "Read and write Unifed Font Object files. (MFEQ fork)"
```

Comment:
Not yet!

Reply:
Sorry, it seems that I mixed up the branches. This was unintentional.